### PR TITLE
Chip press elevation

### PR DESCRIPTION
**Pull request by jslavitz**
Adds love (test) to #18779

### CODE REVIEW

**Comment by lukaspili** [comment]
Thank you! Starting to process all my pending notifications and and I would not be able to add tests before some time.

File: packages/flutter/lib/src/material/chip.dart [hunk 1]
```diff
@@ -1220,10 +1250,11 @@ class _RawChipState extends State<RawChip> with TickerProviderStateMixin<RawChip
   Animation<double> enableAnimation;
   Animation<double> selectionFade;
 
-  static final Animatable<double> pressedShadowTween = Tween<double>(
-    begin: 0.0,
-    end: _kPressElevation,
-  );
+
+//  static final Animatable<double> pressedShadowTween = Tween<double>(
```
Comment (jonahwilliams) [inline packages/flutter/lib/src/material/chip.dart]: What was this used for?

File: packages/flutter/test/material/chip_test.dart [hunk 2]
```diff
@@ -1403,6 +1403,44 @@ void main() {
     expect(deleted, true);
   });
 
+  testWidgets('Chip elevation works correctly', (WidgetTester tester) async {
```
Comment (jonahwilliams) [inline packages/flutter/test/material/chip_test.dart]: Could you do a simplified version of https://github.com/flutter/flutter/blob/master/packages/flutter/test/material/material_test.dart#L123

Just to check that the RenderObject gets the correct shadow?

File: packages/flutter/lib/src/material/chip.dart (hunk 1 above)
Comment (jslavitz) [reply packages/flutter/lib/src/material/chip.dart]: rip, that was there before I started working on it.

File: packages/flutter/test/material/chip_test.dart (hunk 2 above)
Comment (jslavitz) [reply packages/flutter/test/material/chip_test.dart]: Looks like no :(

**Review by jonahwilliams — APPROVED** [review]
LGTM!

**Pull request merged**